backend/backend/settings/develop.py

```diff
@@ -1 +1,29 @@
-from backend.backend.settings.base import *
\ No newline at end of file
+from backend.backend.settings.base import *
+
+FRONTEND_DEV = True
```

Comment:
Remove this and next line, because you don't need this statement anymore, you are in dev file now